aten/src/ATen/function_wrapper.py

```diff
@@ -1386,11 +1386,7 @@ def resize_arg(arg):
         if isinstance(resize, str):
             return "{}.resize_({}.sizes());".format(arg['name'], resize)
         else:
-            resize_scalar = arg.get('resize_scalar', False)
```

Comment:
The existence of `resize_scalar` makes it seem like ger, at some point in the past, supported accepting a 0D tensor. It doesn't accept supporting 0D tensors on master and no one has complained about it, so this change seems fine to me.

If we want to be really safe we can try to figure out if `torch.ger` accepted a 0D tensor at any time in the past and figure out when `torch.ger` stopped accepting a 0D tensor, if at all.

Reply:
Checked a little bit. I think people who made the _th_ger change wanted to make this resize safe.
In legacy code, the size check is inside addr function, but this resize happens before calling addr. _th_ger call addr underline, and addr doesn't allow 0D vec. So ger doesn't support 0D vec anyway